src/solutions/manufacturing.njk

```diff
@@ -135,7 +135,7 @@ description:
             </div>
         </div>
     </div>
-    <div class="mt-20 content max-w-screen-lg m-auto text-center border-2 rounded-lg py-6 px-20 bg-gray-600 bg-opacity-50 border-gray-500">
+    <div class="mt-20 content max-w-screen-lg m-auto text-center border-2 rounded-lg py-6 px-20 bg-gray-600/50 border-gray-500">
```

Comment:
Out of curiosity, why only on manufacturing and data integration and not in the device management page?

Reply:
No reason!